.github/workflows/test.yml

```diff
@@ -2,28 +2,50 @@ name: test
 on:
 - pull_request
 jobs:
-  console-kit_xenial:
-    container: 
-      image: vapor/swift:5.2-xenial-ci
+  console-kit_linux:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        image: ['swift:5.2-xenial', 'swift:5.2-bionic',
+                'swiftlang/swift:nightly-5.2-xenial', 'swiftlang/swift:nightly-5.2-bionic',
```

Comment:
I'd prefer YML style arrays here if possible. Less syntactical bloat.

Reply:
GitHub claims to prefer the flow collection syntax over block collection for matrix variables, though I suspect that's just a matter of their left hand not knowing that their right hand exists, they're worse with YAML parsers in Actions than Apple is at XML parsers in the iOS kernel. (Yeah, I went there!) I could remove the quotes if you feel better about it that way - they're not necessary in this array as far as I recall.

Reply:
I'm using traditional yml arrays here and it's working fine: https://github.com/vapor/fluent-postgres-driver/blob/gm/.github/workflows/test.yml#L13-L27

Pros of this method are:
- Better git diffs
- Room for comments
- No line wrapping

Reply:
I didn't say it didn't _work_, just that GH claims to like it better 😛